src/main/scala/io/iohk/ethereum/vm/WorldStateProxy.scala

```diff
@@ -14,7 +14,7 @@ import io.iohk.ethereum.rlp.RLPList
   * should be kept in memory and applied only after a transaction completes without errors. This does not forbid mutable
   * caches for DB retrieval operations.
   */
-trait WorldStateProxy[WS <: WorldStateProxy[WS, S], S <: Storage[S]] {
+trait WorldStateProxy[WS <: WorldStateProxy[WS, S], S <: Storage[S]] { self: WS =>
```

Comment:
It was needed so that `this: WS` in the `transfer` function. If compilation failed due to: